### ISSUE TITLE
Move dataset "rlp" from IPT to Artsobs

### ISSUE THREAD

**Issue by dagendresen** [gbif-norway/helpdesk#31]
Request from Sabima to move the dataset "rlp" (?) from IPT to Artsobs.

Might perhaps refer to these four datasets??
* https://ipt.gbif.no/resource?r=rlp_b
* https://ipt.gbif.no/resource?r=rlp_f
* https://ipt.gbif.no/resource?r=rlp_l
* https://ipt.gbif.no/resource?r=rlp_v

**Comment by rukayaj** [comment]
Yes I also wasn't sure which dataset it was, thought you might know! I think it must be only one dataset, but not sure...

**Comment by dagendresen** [comment]
I guess it might be one of these. Most likely https://ipt.gbif.no/resource?r=rlp_v
However, the best would be to move ALL four :-)

**Comment by dagendresen** [comment]
Download links
bryophytes -- https://ipt.gbif.no/archive.do?r=rlp_b
fungi -- https://ipt.gbif.no/archive.do?r=rlp_f
lichen -- https://ipt.gbif.no/archive.do?r=rlp_l
vascular -- https://ipt.gbif.no/archive.do?r=rlp_v

DOI dataset PIDs
bryophytes --  https://doi.org/10.15468/1k0gjq
fungi -- https://doi.org/10.15468/kyeeuz
lichen -- https://doi.org/10.15468/z8kzwf
vascular -- https://doi.org/10.15468/nema2n

**Comment by dagendresen** [comment]
I believe we wait for confirmation from NBIC before maybe starting to move the dataset (datasets?). Issue to keep in mind is ensuring that the GBIF occurrenceKey is maintained. (and maybe preserve the occurrenceID? however, persistent urn:uuid:UUIDs might be preferred to the current DwC-triplets?)

**Comment by rukayaj** [comment]
Do we need to do anything for this confirmation from NBIC, like prompt them or something?
